spec/v2/payload_sanitizer_spec.rb

```diff
@@ -324,6 +325,12 @@ class BlankValuesPayloadSanitizer < InputSanitizer::V2::PayloadSanitizer
           sanitizer.errors.length.should eq(2)
           sanitizer.errors.map(&:field).should contain_exactly('/address/city', '/address/zip')
         end
+
+        it "sanitizes to empty hash when given a nil with `allow_nil` flag" do
+          @params = { :nullable_address => nil }
+          sanitizer.should be_valid
+          sanitizer.cleaned[:nullable_address].should eq({})
```

Comment:
I would expect that with this option I get `nil` as `nullable_address` value.
I would change it to nil or at least document this behavior in README.